lint/schema.go

```diff
@@ -34,7 +35,8 @@ const confSchema = `{
 				"type": "integer"
 			},
 			"type": {
-				"type": "string"
+				"type": "string",
+				"enum": ["", "redis"]
```

Comment:
Does adding `enum` automatically makes field "required"? Asking because curious why you allowing empty string here.

Reply:
The default config has this as empty, and we handle empty as default (redis), hence the possible value here.